storage/innobase/xtrabackup/doc/source/xbcloud/xbcloud.rst

```diff
@@ -28,40 +35,237 @@ Version specific information
 - 2.3.1-beta1 - :option:`xbcloud --swift-url` option has been renamed to
   :option:`xbcloud --swift-auth-url`
 
-Usage
---------------------------------------------------------------------------------
+Supported Cloud Storage Types
+================================================================================
+
+In addition to Swift, which has been the only option for storing backups in a
+cloud storage until |Percona XtraBackup| 2.4.14, |xbcloud| supports *Amazon
+S3*, *MinIO*, and *Google Cloud Storage*.
```

Comment:
other S3-compatible storage providers should also work (like Wasabi, Digital Ocean spaces, etc...)

Reply:
We can just say something like,  "other S3 compatible storage implementations may also work but are not officially supported at this time"